lms/envs/common.py

```diff
@@ -2326,6 +2311,22 @@ def _make_locale_paths(settings):  # pylint: disable=missing-function-docstring
 CELERY_BROKER_USER = 'celery'
 CELERY_BROKER_PASSWORD = 'celery'
 
+############################## HEARTBEAT ######################################
+
+# Checks run in normal mode by the heartbeat djangoapp
+HEARTBEAT_CHECKS = [
+    'openedx.core.djangoapps.heartbeat.default_checks.check_modulestore',
+    'openedx.core.djangoapps.heartbeat.default_checks.check_database',
+]
+
+# Other checks to run by default in "extended"/heavy mode
+HEARTBEAT_EXTENDED_CHECKS = (
+    'openedx.core.djangoapps.heartbeat.default_checks.check_celery',
+)
+
+HEARTBEAT_CELERY_TIMEOUT = 5
+HEARTBEAT_CELERY_ROUTING_KEY = HIGH_PRIORITY_QUEUE
```

Comment:
One more last thing (sorry), is there a reason you change the pre-existing default value?  By default, the code had been using `CELERY_DEFAULT_ROUTING_KEY` for this (which defaults to the `CELERY_DEFAULT_QUEUE`).  To be on the safe side (unless it was intentional), let's change the value of this to `CELERY_DEFAULT_ROUTING_KEY`.

Reply:
No worries @nasthagiri, this change was intentional. The original intent behind this PR was to change the heartbeat queue to the high priority. We ran into problems when running grade reports or other long running tasks since all default priority workers were busy, so the celery heartbeat check would time out and trigger false positives in our monitoring infrastructure. Moving this heartbeat check to the high priority queue prevents that since there's generally a high priority worker available.

Reply:
Got it. Thank you.